### ISSUE TITLE
Mention limitations per provider

### ISSUE THREAD

**Issue by tj** [apex/up#146]
AWS request/response limits etc, lack of websockets and so on, so people aren't surprised.

**Comment by olivoil** [comment]
you can use AWS IOT for websockets (mqtt over websockets), there are some limitations on the size of messages, but it scales well, you can connect from browsers or mobile devices, and it can trigger lambda functions

**Comment by tj** [comment]
yeah not sure if I'll go down that road or not, since you'd need two "apps", maybe! probably will omit that for now, I imagine the "container native" stuff is just going to become more and more commoditized, should be a non-problem at some point
